pkg/protocol/engine/consensus/epochgadget/epochorchestrator/performance/snapshot.go

```diff
@@ -0,0 +1,385 @@
+		}
+	}
+
+	return nil
+}
+
+func (t *Tracker) importCommittees(reader io.ReadSeeker) error {
+	var epochCount uint64
+	if err := binary.Read(reader, binary.LittleEndian, &epochCount); err != nil {
+		return errors.Wrap(err, "unable to read committees epoch count")
+	}
+	for i := uint64(0); i < epochCount; i++ {
+		var epoch iotago.EpochIndex
+		if err := binary.Read(reader, binary.LittleEndian, &epoch); err != nil {
+			return errors.Wrap(err, "unable to read epoch index")
+		}
+
+		committee := account.NewAccounts()
+		if err := committee.FromReader(reader); err != nil {
+			return errors.Wrap(err, "unable to read committee")
+		}
+
+		err := t.committeeStore.Set(epoch.Bytes(), lo.PanicOnErr(committee.Bytes()))
+		if err != nil {
+			return errors.Wrap(err, "unable to store committee")
+		}
+	}
+
+	return nil
+}
+
+func (t *Tracker) exportPerformanceFactor(pWriter *utils.PositionedWriter, startSlot, targetSlot iotago.SlotIndex) error {
+	t.performanceFactorsMutex.RLock()
+	defer t.performanceFactorsMutex.RUnlock()
+
+	var slotCount uint64
+	if err := pWriter.WriteValue("pf slot count", slotCount, true); err != nil {
+		return errors.Wrap(err, "unable to write pf slot count")
+	}
+
+	for currentSlot := startSlot; currentSlot <= targetSlot; currentSlot++ {
+		if err := pWriter.WriteValue("slot index", currentSlot); err != nil {
+			return errors.Wrap(err, "unable to write slot index")
+		}
+
+		var accountsCount uint64
+		if err := pWriter.WriteValue("pf account count", accountsCount, true); err != nil {
+			return errors.Wrap(err, "unable to write pf accounts count")
+		}
+
+		if err := t.performanceFactorsFunc(currentSlot).ForEachPerformanceFactor(func(accountID iotago.AccountID, pf uint64) error {
+			if err := pWriter.WriteValue("account id", accountID); err != nil {
+				return errors.Wrap(err, "unable to write account id")
+			}
+			if err := pWriter.WriteValue("performance factor", pf); err != nil {
+				return errors.Wrap(err, "unable to write performance factor")
+			}
+			accountsCount++
+
+			return nil
+		}); err != nil {
+			return errors.Wrap(err, "unable to write performance factors")
+		}
+
+		if err := pWriter.WriteValueAtBookmark("pf account count", accountsCount); err != nil {
+			return errors.Wrap(err, "unable to write pf accounts count")
+		}
+
+		slotCount++
+	}
+
+	if err := pWriter.WriteValueAtBookmark("pf slot count", slotCount); err != nil {
+		return errors.Wrap(err, "unable to write pf slot count")
+	}
+
+	return nil
+}
+
+func (t *Tracker) exportPoolRewards(pWriter *utils.PositionedWriter, targetEpoch iotago.EpochIndex) error {
+	// export all stored pools
+	// in theory we could save the epoch count only once, because stats and rewards should be the same length
+	var epochCount uint64
+	if err := pWriter.WriteValue("pool rewards epoch count", epochCount, true); err != nil {
+		return errors.Wrap(err, "unable to write epoch count")
+	}
+	for epoch := targetEpoch; epoch > iotago.EpochIndex(0); epoch-- {
```

Comment:
What happens if we prune later?

Reply:
then we we need to adjust it later, for now it is alright, I added the TODO